assets/js/phoenix_live_view.js

```diff
@@ -514,11 +514,32 @@ export class LiveSocket {
     for(let type of ["change", "input"]){
       this.on(type, e => {
         let input = e.target
-        let key = input.type === "checkbox" ? "checked" : "value"
-        if(this.prevInput === input && this.prevValue === input[key]){ return }
+
+        let value
+        if (input.type === "checkbox") {
+          value = input["checked"]
+        }
+        else if (input.type === "select-multiple") {
+          let selectedOptions = input["selectedOptions"]
+          const lastItemIndex = selectedOptions.length - 1;
+          // Convert to a string for easier array comparison
+          value = "["
```

Comment:
Is it ok for this key checker to be a mangled, perhaps unidentifiable string?  Or does it have to be properly put together?

Reply:
In the least, I wanted comma separation to ensure true value comparison. The square brackets aren't necessary for the comparison but it's an easier value to inspect.

Reply:
Sounds good!

Looks like this is ok to merge.  What do you think?

Reply:
Yep. @chrismccord, good to merge?